exercises/practice/intergalactic-transmission/intergalactic_transmission.h

```diff
@@ -0,0 +1,13 @@
+#ifndef INTERGALACTIC_TRANSMISSION_H
+#define INTERGALACTIC_TRANSMISSION_H
+
+#include <stdint.h>
+
+#define WRONG_PARITY -1
+
+int transmit_sequence(uint8_t *buffer, const uint8_t *message,
```

Comment:
The focus of this exercise is on the encoding/decoding logic and not so much on figuring out function signatures. As a student I always find it annoying having to figure those out myself when i actually want to focus on the problem at hand, so I kept them here. Let me know if you think they should be removed.